### DISCUSSION TITLE
How to detect/sniff if mobile device?

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#6244] [Help / Questions] [answered]
Please see this lesson for teaching creating games on ccn's new free form programming language:

http://wiki.ccnstudios.com/en/freeform/lessons/Lesson1/lecture_note1 

If you scroll down there is a WebGL Unity application which is a game search engine. 

That WebGL currently does not have the Touch support for mobile screens. 

Therefore I like to load the game interactive in the page as an iframe if&only if desktop browser. 

I know there is a dinky little function I need to add some place, but please help out since it is time-sensitive issue 

Thank you all

DAra

**Comment by NGPixel** [comment, marked as answer]
That's not a Wiki.js-specific issue to be honest.

You can easily find the answer on sites like StackOverflow... Simply check the `navigator.userAgent` property in javascript.

**Comment by darashayda** [comment]
Totally understand, but if some of your drop a quick hint or ref or couple of lines of code, will save us much time. 

Please understand we are going to 200+ 3D Animated characters and a complete programming language API man pages, so any  couple of minutes here and there you could save us, will be much much appreciated. 

In return once we leaned we shall help others here as well. 

This is just a friend comment, it is not meant as a critical statement. 

Thank you again 

Dara
